prow/config.yaml

```diff
@@ -5231,6 +5231,84 @@ postsubmits:
             requests:
               memory: "6Gi"
 
+  - name: ci-kubernetes-bazel-build-1-10
+    agent: kubernetes
+    branches:
+    - release-1.10
+    labels:
+      preset-service-account: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/bazelbuild:v20180201-0184a54dc-0.8.1
+        args:
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/logs"
+        - "--" # end bootstrap args, scenario args below
+        - "--build=//... -//vendor/..."
+        - "--release=//build/release-tars"
+        env:
+        - name: TEST_TMPDIR
+          value: /root/.cache/bazel
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: cache-ssd
+          mountPath: /root/.cache
+        ports:
+        - containerPort: 9999
+          hostPort: 9999
+        resources:
+          requests:
+            memory: "6Gi"
+      volumes:
+      - name: cache-ssd
+        hostPath:
+          path: /mnt/disks/ssd0
+    run_after_success:
+    - name: ci-kubernetes-bazel-test-1-10
+      agent: kubernetes
+      labels:
+        preset-service-account: true
+      spec:
+        containers:
+        - image: gcr.io/k8s-testimages/bazelbuild:v20180201-0184a54dc-0.8.1
```

Comment:
kubernetes 1.10 should be on 0.10.0 or now after the previously mentioned bump goes in 0.11, these jobs have been migrated to `kubekins-e2e` images for release-1.10+ so they share the same base image as the e2es that build with bazel already, see the updated master branch jobs for images/tags

Reply:
this bump went in, the bazel jobs should probably have their config re-copied from the current master-branch configs